### PR TITLE
Create routes and page organization for our project

### PR DESCRIPTION
**Pull request by evadb**
screen shot 2018-03-26 at 10 04 12 am

This PR strips out the pages & routes that we won't be using for our dashboard & reorganizes the pages and routes to fit our needs. Based on the wireframes created by @samsadasivan, we will have the following pages:

index: shows results by agencies
domains: shows results by domains
about: what it says on the tin
feedback: also self explanatory

This is purely an organizational PR

### CODE REVIEW

File: app/views.py [hunk 1]
```diff
@@ -7,18 +7,22 @@
 
 def register(app):
 
-    @app.route("/data/")
-    def data():
-        return render_template("data.html")
-
     @app.route("/")
     def index():
         return render_template("index.html")
```
Comment (buckley-w-david) [inline app/views.py]: Is it intentional that this displays the agencies page?

File: templates/index.html [hunk 2]
```diff
@@ -1,31 +1,39 @@
...
-      ></div>
-    	{% include 'includes/analytics/donut.html' %}
-      <a href="/analytics/domains/" class="button">View Analytics Results</a>
-    </div>
+{% include 'includes/https/header.html' %}
+
+<section id="main-content" class="white-bkg">
+
+  <nav class="tab-menu" role="tablist">
+    <a href="/https/domains/" class="tab" role="tab" aria-controls="tab-panel" aria-selected="false">By Domain</a>
```
Comment (buckley-w-david) [inline templates/index.html]: `/https/domains` is no longer a valid route, no?

File: app/views.py (hunk 1 above)
Comment (evadb) [reply app/views.py]: Yes, based on the wireframe the index page should land users on the agencies list

File: templates/index.html (hunk 2 above)
Comment (evadb) [reply templates/index.html]: I strive to make the website a terrible, broken experience

**Pull request merged**